test/reducers/columnStateHelper-test.js

```diff
@@ -180,15 +180,23 @@ describe('columnStateHelper', function() {
         isColumnResizing: true,
       };
 
+      const fakeElementTemplates = {
+        cell: [],
+        footer: [],
+        groupHeader: [],
+        header: []
+      };
```

Comment:
Ya that's fine.  It would be more ideal if we added some things here, but since these hold React elements they're harder to fake.